Makefile

```diff
@@ -120,7 +120,7 @@ minikube-start:
 		--container-runtime=containerd \
 		--memory=${minikube-memory} \
 		--cpus=4 \
-		--kubernetes-version=1.21.1 \
+		--kubernetes-version=1.19.14 \
```

Comment:
The behavior of pods changed between 1.19 and 1.21. In the former, pods that exceed their deadline become Failed with a reason of "DeadlineExceeded" (which we catch). In the latter, pods simply become Completed, and aren't in a failed state

Reply:
Both are working, we handle both behaviors